pkg/migrate/cstor/migrationtask.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migrate
+
+import (
+	v1Alpha1API "github.com/openebs/api/v2/pkg/apis/openebs.io/v1alpha1"
```

Comment:
will there be any lint error due to camel case import alias used here ?

Reply:
No tested the linter